web/src/pages/core/public/PrivacyPolicy/modal.jsx

```diff
@@ -0,0 +1,25 @@
+import './modal.css';
```

Comment:
Material-UI has there own modal component which will work alot more smoothly and require less work.